lnrpc/walletrpc/walletkit.proto

```diff
@@ -220,46 +220,54 @@ message BumpFeeRequest {
 message BumpFeeResponse {
 }
 
+message SharedKeyRequest {
+    bytes ephemeral_pubkey = 1;
+}
+
+message SharedKeyResponse {
+    bytes shared_key = 1;
+}
+
 service WalletKit {
     /**
     DeriveNextKey attempts to derive the *next* key within the key family
     (account in BIP43) specified. This method should return the next external
     child within this branch.
     */
-    rpc DeriveNextKey(KeyReq) returns (signrpc.KeyDescriptor);
+    rpc DeriveNextKey (KeyReq) returns (signrpc.KeyDescriptor);
```

Comment:
Do those spaces come from some kind of proto cleaner tool?

Reply:
Yes, the formatter of my IDE has support for proto files. But we probably want this in a separate PR, most of the proto files have inconsistent formatting.

Reply:
I don't mind